src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/HtmlCSharp/DefaultLSPRequestInvoker.cs

```diff
@@ -80,6 +80,11 @@ public DefaultLSPRequestInvoker(ILanguageClientBroker languageClientBroker)
 
         private async Task<TOut> RequestServerCoreAsync<TIn, TOut>(MethodInfo lspPlatformMethod, string method, LanguageServerKind serverKind, TIn parameters, CancellationToken cancellationToken)
         {
+            if (lspPlatformMethod is null)
```

Comment:
I ran into a nulref on this while I was testing it because I had an old VS. Thought I might add an explicit guard just in case.